game.py

```diff
@@ -20,7 +20,38 @@ def snowman(snowman_word):
     If the player wins and, 
     'Sorry, you lose! The word was {snowman_word}' if the player loses
     """
-    pass
+    correct_letter_guess_statuses = build_letter_status_dict(snowman_word)
+    wrong_guesses_list = []
+    user_wins = is_word_guessed(snowman_word, correct_letter_guess_statuses)
+
+    while (
+        (len(wrong_guesses_list) < SNOWMAN_MAX_WRONG_GUESSES)
+        and (not user_wins)
+    ):
+        user_input = get_letter_from_user(correct_letter_guess_statuses, 
+                                        wrong_guesses_list)
```

Comment:
Following the [PEP 8 Style Guide for Indentation](https://peps.python.org/pep-0008/#indentation), *"Continuation lines should align wrapped elements either vertically using Python’s implicit line joining inside parentheses, brackets and braces, or using a hanging indent."*.
```suggestion
        user_input = get_letter_from_user(
            correct_letter_guess_statuses, 
            wrong_guesses_list)
```